src/libxtp/gwbse/gwbse.cc

```diff
@@ -721,5 +724,60 @@ bool GWBSE::Evaluate() {
       << TimeStamp() << " GWBSE calculation finished " << flush;
   return true;
 }
+
+Eigen::MatrixXd GWBSE::PrepareHQP_forBSE(const Eigen::MatrixXd& Hqp) {
```

Comment:
needs a better name than Prepare, maybe Scale or Expand or so. Prepare is pretty generic.

Reply:
Done